script/bump_towncrier.py

```diff
@@ -0,0 +1,38 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""This script updates the path to the news file in the towncrier config file."""
+
+import argparse
+import re
+from pathlib import Path
+
+NEWSFILE_PATTERN = re.compile(r"doc/whatsnew/\d/\d.\d+/index\.rst")
+NEWSFILE_PATH = "doc/whatsnew/{major}/{major}.{minor}/index.rst"
+TOWNCRIER_CONFIG_FILE = Path("towncrier.toml")
+
+
+def main():
```

Comment:
```suggestion
def main() -> None:
```

``mypy --strict`` will complain otherwise.

Reply:
Changed.